perceptia-one/perceptia/src/components/QuizGallery.js

```diff
@@ -34,7 +39,8 @@ class QuizGallery extends React.Component {
     }
 
     componentDidMount() {
-        fetch('http://students.washington.edu/long27km/any_quiz/v1/list')
+    
+        fetch(constants.localhost.url + 'list')
```

Comment:
See comment in constants.js about using the api.url constant.

Reply:
Fixed